src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
Don't use verbs for class/interface names. It's better to name the interface something that represents a property or capability, like `AreaCalculator`.